src/components/App.js

```diff
@@ -39,21 +40,20 @@ class App extends React.Component {
     }
   }
   render() {
-    if (this.props.currentUser) {
-      return (
-        <div>
-          <LoggedOutView appName={this.props.appName} />
-          {this.props.children}
-        </div>
-      );
-    } else {
-      return (
-        <div>
-          <LoggedInView appName={this.props.appName} />
-          {this.props.children}
-        </div>
-      );
-    }
+    //  if (this.props.currentUser) {
+    return (
+      <div>
```

Comment:
You can use react fragment here rather than `div` that will show-up in the dom tree